CMakeLists.txt

```diff
@@ -116,7 +116,12 @@ else()
 endif()
 
 # Find Ignition Gazebo
-find_package(ignition-gazebo3 COMPONENTS all)
+if(NOT CMAKE_BUILD_TYPE STREQUAL "PyPI")
+    find_package(ignition-gazebo3)
+else()
+    find_package(ignition-gazebo3 REQUIRED)
+endif()
+
```

Comment:
Why is this necessary?

Reply:
Today during the final polishing I found my setup in the situation where I wanted to generate a PyPI package but CMake did not find Ignition Gazebo. Without this if statement, the generation of the package (launched by setuptools from `setup.py`) will succeed. In fact, in this case the CMake project would not fail and I was obtaining a PyPI package with just the Python files.

This is very dangerous in the case of a bugged Docker image used for packaging (remember that it is autogenerated with cron weekly builds). It would result to a uncomplete package pushed to PyPI.